macros/SpecData.json

```diff
@@ -1431,7 +1431,12 @@
   },
   "WebDriver": {
     "name": "WebDriver",
-    "url": "https://w3c.github.io/webdriver/webdriver-spec.html",
+    "url": "https://w3c.github.io/webdriver/",
+    "status": "Living"
+  },
+  "WebDriver Level 1": {
```

Comment:
This is just <q>WebDriver</q>.

Reply:
There’s a difference between https://www.w3.org/TR/webdriver1/ and https://w3c.github.io/webdriver/, and it’s useful to be able to reference both.

The former recommendation is a timestamped dump for legal reasons and does not reflect the reality of existing implementations. The latter is an evolving document that accurately tracks and reflects reality in WebDriver implementations.

Reply:
I’m not aware of anyone referencing level 1 on MDN.

Reply:
Then we shouldn't add this here so that people _can't_ reference it. Or am I misunderstanding something?

Reply:
I thought this was a bibliography of specifications you could reference, like https://www.specref.org/, but if you want to limit the selection for MDN I can remove the level 1 entry?

Reply:
This spec list is not meant to be exhaustive like specref. My understanding is that it only collects the specs that can and should be referenced from MDN pages. We have removed dated specs in the past. See https://github.com/mdn/kumascript/pull/284 and https://github.com/mdn/kumascript/pull/220 for previous efforts to get rid of outdated specs. I'd like to avoid the situation in which we have to edit lots of wiki pages to remove dated specs by not allowing them in the first place.

So, yes, imo, we should remove the level 1 entry.